src/WebClient.ts

```diff
@@ -122,6 +122,11 @@ export class WebClient extends EventEmitter {
    */
   private logger: Logger;
 
+  /**
+   * Static headers on every request.
+   */
+  private staticHeaders?: object;
```

Comment:
I'm curious if there is any purpose of even making it a property on the WebClient? I only use is in the constructor when the axios instance is created.

Reply:
IMHO i don't think we should keep this property. i think you've got the right idea about mixing the object provided in the options into the axios instance inside the constructor, and that's all we need right now.